tests/decodeinput.cpp

```diff
@@ -45,11 +45,15 @@ class MyDecodeInput : public DecodeInput{
     virtual bool isEOS() {return m_parseToEOS;}
     virtual bool init() = 0;
     virtual const string& getCodecData();
+    uint16_t getWidth() {return m_width;}
```

Comment:
could we move this two functions to DecodeInput and remove similar functions in DecodeInputAvFormat